assignments/objects.js

```diff
@@ -49,16 +88,37 @@ const example = {
 // 3. Nest a grandchild object in the child object with properties for name and age.  The name will be Sam and the age will be 30
 // 4. Give each of the objects the ability to speak their names using the this keyword.
 
-const parent = {}
+const parent = {
+  "name": "Susan",
+  "age": 70,
+  const child = {
+    "name": "George",
+    "age": 50,
+    const grandchild = {
+      "name": "Sam",
+      "age": 30,
+    }
+  }
+}
 
 // Log the parent object's name
-
+console.log(parent.name);
 // Log the child's age
-
+console.log(child.age);
 // Log the name and age of the grandchild
-
+console.log(grandchild.name, grandchild.age);
 // Have the parent speak
-
+parent.speak = function(){
+  console.log("Hello, I am the parent!");
+}
+console.log(parent.speak());
 // Have the child speak
-
+child.speak = function(){
+  console.log("Hello, I am the child!");
+}
+console.log(child.speak());
```

Comment:
The idea here is for the child object nested inside the parent object to have its own `speak` method. Thus, its speak method would be invoked as `parent.child.speak()`.